TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java

```diff
@@ -101,7 +101,7 @@ public static void rotation(final CommandContext cmd, CommandSender sender) thro
     }
 
 
-    @Command(aliases = {"cycle"}, desc = "Cycle to a new map.")
+    @Command(aliases = {"cycle", "skip"}, desc = "Cycle to a new map.")
```

Comment:
Well, the title is misleading, That's not `/next`, that's `/skip`

Reply:
my mistake, apologies. Changed